app/third_party/twitter.rb

```diff
@@ -0,0 +1,12 @@
+class Twitter
+  URL = 'https://api.twitter.com/1.1'
+
+  def initialize(user)
+    @user = user
+    @token = user.twitter_token
+  end
+
+  def timeline
```

Comment:
Same as above, create a TwitterService and define the method, use query params instead of hardcoding the method url, a good practice is to define the method like: 
```
  def self.get_timeline
    self.get("/statuses/user_timeline", user_params).body
  end

  def user_params_with_token
    {
      query: {
        user_id: user.twitter_id
        token: user.twitter_token
    }
  end
```